crates/ruff_linter/src/rules/pylint/rules/useless_import_alias.rs

```diff
@@ -53,6 +54,75 @@ pub(crate) fn useless_import_alias(checker: &mut Checker, alias: &Alias) {
         return;
     }
 
+    // While the fix is specified as always available,
+    // the presence of a user-specified required import (I002)
+    // with a useless alias causes an infinite loop. So
+    // in this case we keep the diagnostic but omit the fix.
+    // See https://github.com/astral-sh/ruff/issues/14283
```

Comment:
I don't think our testing framework will accept that an always fixable fix isn't always fixable, which makes sense.

Reply:
Of course you're right! Changed to "Sometimes" with special message/fix title emitted in case of conflict.